### PR TITLE
Automatic cleanup of stale peers

### PR DESCRIPTION
**Pull request by awlx**
This adds a long wanted feature to automatically cleanup stale peers. Thanks to @niyawe

### CODE REVIEW

**Comment by awlx** [comment]
This needs testing on our setup.

File: wgkex/worker/app.py [hunk 1]
```diff
@@ -21,6 +30,11 @@ def main():
     domains = config.load_config().get("domains")
     if not domains:
         raise DomainsNotInConfig("Could not locate domains in configuration.")
+    clean_up_threads = []
+    for domain in domains:
+        thread = threading.Thread(target=clean_up_worker, args=(domain[2:],))
```
Comment (niyawe) [inline wgkex/worker/app.py:35]: You probably want to change the domain argument here ;)

Comment (awlx) [reply wgkex/worker/app.py:35]: Yep :D. I was just testing it. Thanks for the hint.

**Comment by awlx** [comment]
Works. Closes https://github.com/freifunkMUC/wgkex/issues/48

**Pull request merged**